virtcontainers/qemu_ppc64le.go

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2018 IBM
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"encoding/hex"
+	"fmt"
+	"os"
+	"strconv"
+
+	govmmQemu "github.com/intel/govmm/qemu"
+)
+
+type qemuPPC64le struct {
+	// inherit from qemuArchBase, overwrite methods if needed
+	qemuArchBase
+}
+
+const defaultQemuPath = "/usr/libexec/qemu-kvm"
+
+const defaultQemuMachineType = QemuPPC64le
+
+const defaultQemuMachineOptions = "accel=kvm,usb=off"
+
+const defaultPCBridgeBus = "pci.0"
+
+var qemuPaths = map[string]string{
+	QemuPPC64le: defaultQemuPath,
+}
+
+var kernelParams = []Param{
```

Comment:
A lot of these duplicate with kernelParams in qemu_amd64.go, which means we should define a kernelParamsBase in qemu_arch_base.go.